content/codespaces/reference/disaster-recovery-for-github-codespaces.md

```diff
@@ -38,7 +38,7 @@ You can check the current service status on the [Status Dashboard](https://www.g
 
 ## Option 3: Clone the repository locally or edit in the browser
 
-While {% data variables.product.prodname_github_codespaces %} provides the benefit of a pre-configured developer environmnent, your source code should always be accessible through the repository hosted on {% data variables.product.prodname_dotcom_the_website %}. In the event of a {% data variables.product.prodname_github_codespaces %} outage, you can still clone the repository locally or edit files in the {% data variables.product.company_short %} browser editor. For more information, see "[AUTOTITLE](/repositories/working-with-files/managing-files/editing-files)."
+While {% data variables.product.prodname_github_codespaces %} provides the benefit of a pre-configured developer environment, your source code should always be accessible through the repository hosted on {% data variables.product.prodname_dotcom_the_website %}. In the event of a {% data variables.product.prodname_github_codespaces %} outage, you can still clone the repository locally or edit files in the {% data variables.product.company_short %} browser editor. For more information, see "[AUTOTITLE](/repositories/working-with-files/managing-files/editing-files)."
```

Comment:
#### Before
https://docs.github.com/en/codespaces/reference/disaster-recovery-for-github-codespaces#option-3-clone-the-repository-locally-or-edit-in-the-browser
<img width="722" alt="image" src="https://github.com/github/docs/assets/2119212/01ee6ec3-7c05-4f07-b561-cd4618743bb3">

#### After
https://docs-30890-94eb78.preview.ghdocs.com/en/codespaces/reference/disaster-recovery-for-github-codespaces#option-3-clone-the-repository-locally-or-edit-in-the-browser
<img width="717" alt="image" src="https://github.com/github/docs/assets/2119212/8dfce313-dce4-43b6-b683-87ba8ed25496">